### ISSUE TITLE
代码和公式的对应问题，期待您的解答

### ISSUE THREAD

**Issue by ZhuLiangyu123** [wangsen99/GEFU#4]
您好，在您train主函数代码中，反射图一致性损失的代码是：loss_retx_a = F.l1_loss(cyc_rec_a_R, R_a_gt.detach()) + (1. - ssim(cyc_rec_a_R, R_a_gt.detach())) + net_enh.enhance_loss(cyc_fake_b_R, cyc_rec_a_R, img_a, I_a.detach())，论文中公式（5）对应的公式应该是loss_retx_a = F.l1_loss(cyc_rec_a_R, R_a_gt.detach()) + net_enh.enhance_loss(cyc_fake_b_R, cyc_rec_a_R, img_a, I_a.detach())，中间的(1. - ssim(cyc_rec_a_R, R_a_gt.detach()))在公式中没有找到。请问(1. - ssim(cyc_rec_a_R, R_a_gt.detach()))这行代码是否有必要？或者是我没有完全理解您的论文及公式（5），期待您的回复，谢谢！

**Comment by wangsen99** [comment]
您好，使用ssim loss 是常见的训练技巧，为了避免论文写作的冗余，我省略了这一项

**Comment by ZhuLiangyu123** [comment]
好的，感谢您百忙之中的回复！
